spec/docs/joinconditions.md

```diff
@@ -96,3 +96,77 @@ then the `rml:child` shortcut could be used.
     rml:logicalSource <LS2> ;
     rml:subjectMap <#SM2> .
 ```
+## Join types
```

Comment:
I would prefer not to call this Join types. I also don't see the need for a separate section, since we're essentially still talking about join conditions.

Reply:
ok for me, adapted accordingly